server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -197,6 +198,34 @@ func (r *statsRecorder) Enqueue(ctx context.Context, invocation *inpb.Invocation
 	}
 }
 
+func scoreCardBlobName(invocationID string) string {
+	blobFileName := invocationID + "-scorecard.chunk"
```

Comment:
uber nit: since this is a non-chunked blob, consider removing the `.chunk` suffix?

Reply:
Done.